### DISCUSSION TITLE
Tips & advice for students interested in STEM

### DISCUSSION THREAD

**Discussion by Nautman** [digitalungdom-se/digitalungdom.se#102] [General]
For a long time we've discussed how unfortunate it is that not all students know about opportunities or activities they can pursue in their interest of STEM subjects.

For example, I regret that I didn't know I could study mathematics for fun at my local university. (Matematisk cirkel at KTH Royal Institute of Technology).

It'd be really cool if we could start a discussion on what ambitious students can do :)

I've written in English, but feel free to post in Swedish as well!

I will post some questions below to act as threads, but feel free to post whatever you would like wherever in this thread! No bureaucracy required 🚀 

EDIT: If we feel like these tips and advice become comprehensive enough, we could add it to our website. (Hence the location within this repository. However, it could also be spun off to another project!)

EDIT 2: Perhaps in the future we can sort tips, such as top 3 tips for students interested in programming.

**Comment by Nautman** [comment]
What can you do if you're interested in Mathematics?

**Reply by Nautman** [reply]
A well known resource is Khan Academy and is a great resource for understanding various mathematical subjects. I remember that after a lesson I would use Khan Academy. It was basically an extra tutoring session and was very helpful. I think it's also possible to use it if you want to pursue mathematics at a higher level than your school is currently offering. Although, I'm a little unsure if there are too few exercises? 🤔 

Another project that you can do if you want to pursue programming as well, is to use [Project Euler](https://projecteuler.net/)! It has small math problems that you solve by combining mathematics and programming.

In Sweden, KTH and SU offers a study programme for "gymnasie"-students interested in mathematics, called [Matematisk Cirkel](https://www.math-stockholm.se/samverkan/cirkel/matematiska-cirkeln-lasaret-2020-2021-1.1001414). This year, the circle is taught via Zoom.

**Comment by Nautman** [comment]
What can you do if you're interested in Engineering?

**Reply by Nautman** [reply]
A general tip (one that I've seen in UK contexts) is to browse the [i-want-to-study-engineering.org](https://i-want-to-study-engineering.org/) website. I didn't use it all that much though.

I think it's also run by Cambridge University!

**Reply by Nautman** [reply]
I heard about a [Creative Engineering course by Mark Rober](https://monthly.com/mark-rober-engineering). I like Marks content on his YouTube channel a lot, so I believe his course should be interesting!

However, it does cost 249 USD (which will rise to 499 USD in the future). I do believe that most things on this list should be free or affordable to ensure a democratic access to develop your interest in STEM. Since many free time activities cost money (sports, theater, etc) I think these tips should be allowed but perhaps not remain in focus.

Perhaps there are more affordable alternatives and maybe organized by local organizations in Sweden? :)

**Comment by Nautman** [comment]
What can you do if you're interested in Astronomy?

**Reply by Nautman** [reply]
From personal experience, I would highly recommend the Swedish youth organization called [Astronomisk Ungdom](https://astronomiskungdom.se).

They are a very good organization with many ambitious members.

I've also been to their space camp called [Rymdforskarskolan](https://www.astronomiskungdom.se/rymdforskarskolan/)! It is for ambitious students interested in space and currently studying in "gymnasiet" (high-school).

**Comment by Nautman** [comment]
What can you do if you're interested in researching / creating reports?

**Reply by Nautman** [reply]
[Rays for excellence](https://raysforexcellence.se/) is a summer research academy for students in the second year in high school that have a great interest in the natural sciences, technology and mathematics.

[Research science institute](https://www.cee.org/research-science-institute) is a 6-week long summer programme at Massachusetts Institute of Technology (MIT) in Boston, USA. In Sweden, you can [apply via Unga Forskare](https://ungaforskare.se/category/rsi/).

**Reply by charlesmaddock** [reply]
Om man har ett gymnasiearbete kan man alltid ansöka till Utställningen Unga Forskare! (Det är typ en tävling för gymnasiearbeten) 

Jag ansökte och tänkte att jag inte skulle komma någon vart - men sen lyckades jag vinna en stipendieresa till Macao och där lyckades jag också vinna pris, så det är verkligen värt att ansöka! Läs mer här: https://ungaforskare.se/utstallningen/

**Comment by Nautman** [comment]
What can you do if you're interested in technology / programming?

**Reply by charlesmaddock** [reply]
Well, if someone has found this Github discussion I'm quite sure they already know that Digital Ungdom a great organisation for those who are interested in technology / programming! :D

**Reply by Nautman** [reply]
If you want to get a fundamental but thorough understanding for Computer Science, I highly recommend [CS50 by Harvard](https://online-learning.harvard.edu/course/cs50-introduction-computer-science)! It is a free course offered on edX and allows you to gain a verified certificate for 199 USD if you complete the course. The course covers things from abstraction, algorithms, data structures, to security and web development.

Some personal reflections are that people who are self-taught often overestimate their knowledge and skills in Computer Science. I had programmed for a couple of years before I took it, and learned a lot of things that I found very useful. It was a pure joy to listen to the lectures given by @dmalan and I remember being excited to go home and listen to lectures (when I had just gotten spring break!).

There are also some other courses organized by @cs50 such as [CS50's Introduction to Game Development](https://online-learning.harvard.edu/course/cs50s-introduction-game-development), and [CS50's Web Programming with Python and JavaScript](https://online-learning.harvard.edu/course/cs50s-web-programming-python-and-javascript). I can't speak for the quality of those but I know the team holds a high standard.
